docs/pages/admin-guides/management/guides/migrating-iam-ic-from-okta-to-teleport.mdx

```diff
@@ -0,0 +1,325 @@
+
+We will be using a very limited subset of the Teleport Okta integration in this 
+deployment, disabling all features except periodic user synchronization. This
+configuration is not supported by the normal installation UI, so we will have to
+use `tctl` to install it:
+
+```console
+$ ./tctl plugins install okta \
+    --org ${OKTA_ORG_URL} \
+    --saml-connector ${TELEPORT_SAML_CONNECTOR_NAME} \
+    --app-id ${OKTA_SAML_APP_ID} \
+    --api-token ${OKTA_API_TOKEN} \
+    --no-scim \
+    --no-accesslist-sync \
+    --no-appgroup-sync
+```
+
+This will install the Okta integration and start the User sync service. You can
+monitor the state of the Okta integration in the Teleport Integrations UI.
+
+## Step 3: Wait for user sync
+
+To make sure everything is working, wait until the first Okta to Teleport user
+sync has occurred. You can verify this by either
+ - refreshing the user page and finding your Okta users,
+ - checking the Okta integration status page, or
+
+Once your Okta users are imported into Teleport, you can progress to the next
+step.
+
+## Step 4: Install the Teleport Identity Center Integration.
+
+Again, we need to install the plugin using `tctl`. 
+
+```console
+$ ./tctl plugins install awsic \
+    --arn ${IDENTITY_CENTER_INSTANCE_ARN} \
+    --region ${IDENTITY_CENTER_INSTANCE_REGION} \
+    --use-system-credentials \
+    --url ${IDENTITY_CENTER_SCIM_BASE_URL} \
+    --token ${IDENTITY_CENTER_SCIM_BEARER_TOKEN} \
+    --default-owner ${TELEPORT_ACCESS_LIST_DEFAULT_OWNER} \
+    --user-origin okta \
+    --account-name ${ACCOUNT_NAME_ALLOW_FILTER} \
+    --group-name ${GROUP_NAME_ALLOW_FILTER}
+```
+
+During the installation process, Teleport will import all of the Identity Center 
+groups that match its allow list (or all of them, if no allow list is defined)
+and create matching Access Lists, preserving the group membership and account
+assignments.
+
+<Admonition type="warning">
+Individual User Account Assignments will _not_ be preserved during import. You
+will need to ensure these are preserved manually, or converted to group assignments
+prior to installation.
+</Admonition>
+
+### Group import control
+
+The Group import allow-list is controlled by the `--group-name` option. You can
+specify multiple filters and a Group will be imported if matches _any_ of the
+supplied filters. Filters can be either literal names, globbed names or Go
+compatible regular expressions. To treat a filter as a regular expression, 
+enclose it in a leading `^` and trailing `$`.
+
+Example filters:
+ - `administrators`: The literal "administrators" group
+ - `site-*`: Any group with the prefix `site-`
+ - `^(?:[^a]|a[^w]|aw[^s]|aws[^\-]).*$`: Any group that does _not_ have the prefix `aws-`
+
+
+<Admonition type="note">
+Ensure that there is no overlap between the groups imported to Teleport and the
+groups you want Okta to maintain control over.
+</Admonition>
+
+<Admonition type="warning">
```

Comment:
Stacking admonitions like this creates a lot of noise. I would prefer more plain text and only use the admonition when there's something you really need to highlight.